### DISCUSSION TITLE
DFTB+ 21.0

### DISCUSSION THREAD

**Discussion by jjventura** [dftbplus/dftbplus#1129] [General]
good morning, i'm Jorge Joe from Cuba. i try install DFTB+ in my PC but i give me the next problem in the moment of run get_opt_external i don't that to do, you could help me? the error is the next:
*** Getting external: slakos
    into /home/jorge/dftbplus/external/slakos

Download:
    /home/jorge/dftbplus/external/slakos/testparams.tar.gz
    from https://github.com/aradi/testparams/archive/cd7536f659be2665cd78d0297f00d377544840aa.tar.gz
Traceback (most recent call last):
  File "/home/jorge/dftbplus/utils/./get_opt_externals", line 258, in <module>
    main()
  File "/home/jorge/dftbplus/utils/./get_opt_externals", line 99, in main
    getter(destdir)
  File "/home/jorge/dftbplus/utils/./get_opt_externals", line 143, in get_slakos
    download_file(url, fname)
  File "/home/jorge/dftbplus/utils/./get_opt_externals", line 169, in download_file
    sub.check_call(['wget', '-q', '-O', fname, url])
  File "/usr/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['wget', '-q', '-O', '/home/jorge/dftbplus/external/slakos/testparams.tar.gz', 'https://github.com/aradi/testparams/archive/cd7536f659be2665cd78d0297f00d377544840aa.tar.gz']' returned non-zero exit status 4.

**Comment by aradi** [comment]
Can you simply test, what happens if you execute

```
wget -q -O testparams.tar.gz https://github.com/aradi/testparams/archive/cd7536f659be2665cd78d0297f00d377544840aa.tar.gz
```
at your command line?

**Reply by jjventura** [reply]
thank so much, the ploblem it did that the passworld of thr proxy for wget it have be wrong.
